pkg/api/common/versions.go

```diff
@@ -104,7 +104,6 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.12.6":         false,
 	"1.12.7":         true,
 	"1.12.8":         true,
-	"1.12.9":         true,
```

Comment:
Can we re-add it back in as explicit `"false", and with a comment as to why?

Reply:
this was never released so a simple commit revert made most sense to me

Reply:
I kind of mean as a reminder to ourselves so when 1.12.10 arrives we run extra upgrade validation on it to see if things have been fixed

Reply:
how about we keep track of that in an issue? we can put it in the backlog too so we don't forget

Reply:
we also need to do more investigation to understand what's going on... it's still unclear what the exact root cause is.

Reply:
Let's do both!